packages/components/src/components/input/index.tsx

```diff
@@ -7,7 +7,7 @@ export interface InputProps extends React.InputHTMLAttributes<HTMLInputElement>
   message?: string;
 }
 
-export function Input({ severity, message, ...props }: InputProps) {
+export function Input({ severity = 'neutral', message, ...props }: InputProps) {
```

Comment:
introducing a default value would be considered a breaking change 🤔

Reply:
I think it's a patch because:

1. `neutral` should be the default, and equivalent to undefined. I overlooked styling it, so this is kind of a bug.
2. Previously it was green text on white background, not it's a more toned down green text on almost white blueish gray background.
3. I'm using the `Input` once.